distribution/docs/src/main/resources/_contents/_sources/solr-catalog-provider-contents.adoc

```diff
@@ -1,24 +1,15 @@
 
-==== Catalog Solr Embedded Provider
+==== Solr Embedded Provider
 
-The Solr Catalog Embedded Provider is an embedded, local Solr Server instance used in conjunction with an Apache Solr Server data store.
-It is a local instance that is a lightweight Catalog provider solution. However, it does not provide a Solr Admin GUI or a "REST-like HTTP/XML and JSON API." If that is necessary, see Standalone Solr Server.
+The Solr Embedded Provider is an embedded, local Solr Server instance used in conjunction with an Apache Solr Server data store.
+It is a local instance that is a lightweight Catalog provider solution.
+However, it does not provide a Solr Admin GUI or a REST-like HTTP/XML and JSON API.
+If that is necessary, see Standalone Solr Server or Solr Cloud.
```

Comment:
no, it was decided it should be Solr Cloud, i will look through the docs again and make sure it's Solr Cloud